components/blitz/resources/omero/api/IAdmin.ice

```diff
@@ -17,9 +17,6 @@ module omero {
 
     module api {
 
-        /**
-         * See <a href="http://hudson.openmicroscopy.org.uk/job/OMERO/javadoc/ome/api/IAdmin.html">IAdmin.html</a>
```

Comment:
The most permanent URL would be http://downloads.openmicroscopy.org/latest/omero5.1/api/slice2html/omero/api/IAdmin.html. Is there still an `ome.api.*` ?